### PR TITLE
Github username bibekpanda55

### PR DESCRIPTION
**Pull request by bibekpanda55**
Added a math riddle number 9 with title magic number

### CODE REVIEW

**Comment by bibekpanda55** [comment]
@harshptl14 I have pulled again and added the riddle please review my PR and merge it

File: lib/categories/Math/MathRiddlelLists.dart [hunk 1]
```diff
@@ -95,6 +95,15 @@ class _StateMathRiddles extends State<MathRiddles> {
                     Navigator.of(context).push(
                         MaterialPageRoute(builder: (ctx) => Math8HomeScreen()));
                   }),
+              ActionCard(
+                  name: 'Magic Number',
+                  description:
+                      'A math riddle which is very fun to try out by @bibekpanda55',
+                  number: '9',
```
Comment (harshptl14) [inline lib/categories/Math/MathRiddlelLists.dart:102]: Change to 10

File: lib/categories/Math/math9@bibekpanda55/math9HomePage.dart [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    child: Container(
+                      padding: EdgeInsets.all(10),
+                      decoration: BoxDecoration(
+                          border: Border.all(width: 1.6),
+                          color: Colors.transparent),
+                      width: MediaQuery.of(context).size.width * 0.9,
+                      child: Align(
+                        alignment: Alignment.center,
+                        child: Text(
+                          "There is a three digit number.The second digit is four times as big as the third digit, while the first digit is three less than the second digit.What is the number?",
```
Comment (harshptl14) [inline lib/categories/Math/math9@bibekpanda55/math9HomePage.dart:53]: Make it a little bit clear😬

Comment (bibekpanda55) [reply lib/categories/Math/math9@bibekpanda55/math9HomePage.dart:53]: Yeah will do that

**Review by harshptl14 — DISMISSED** [review]
Can you make those two changes, and pull it once again.

File: lib/categories/Math/MathRiddlelLists.dart (hunk 1 above)
Comment (bibekpanda55) [reply lib/categories/Math/MathRiddlelLists.dart:102]: > Can you make those two changes, and pull it once again.

Yes I can

**Pull request closed without merging**